lusidtools/cocoon/cocoon.py

```diff
@@ -418,10 +418,10 @@ def load_portfolio_batch(
 
             sync_batches = [
```

Comment:
Add a comment to explain what this is doing

Reply:
Added